Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.12.0
+FROM alpine
```

Comment:
Can't use no version at all, that's as bad as `latest`

Reply:
My reasoning here was that this image is used for tflint - which is a static binary - so actually everything else is just to get tflint into the image.

Therefore the version doesn’t matter, and might as well be latest so that we don’t have to update.

Arguably, this should be refactored as a multi stage build so that we ONLY end up with the tflint binary in the image, but that seemed wider than my necessary scope.

Reply:
Don't actually have any strong views here so I've pushed an update to tag at alpine 3